src/N3Parser.js

```diff
@@ -866,6 +894,27 @@ export default class N3Parser {
     }
   }
 
+    // ### `_readRDFStarTail` reads the end of a nested RDF* triple
```

Comment:
Do you really mean the deprecated `RDF*`, or should this name the current `RDF-star`?
```suggestion
    // ### `_readRDFStarTail` reads the end of a nested RDF-star triple
```